tests/conftest.py

```diff
@@ -1,16 +1,9 @@
 import pytest
-from prefect.utilities.testing import prefect_test_harness
 
 from prefect_aws import AwsCredentials
 from prefect_aws.client_parameters import AwsClientParameters
 
 
-@pytest.fixture(scope="session", autouse=True)
-def prefect_db():
-    with prefect_test_harness():
-        yield
-
```

Comment:
Why did you remove this fixture?

Reply:
I think it was crashing tests earlier (e.g. `from prefect.utilities.testing import ...`), but now I think it's not used since the tests are still passing?

Reply:
```
asyncio: mode=auto
collected 58 items

tests/test_batch.py .                                                                                                                                                                              [  1%]
tests/test_client_parameters.py ....                                                                                                                                                               [  8%]
tests/test_s3.py ...............                                                                                                                                                                   [ 34%]
tests/test_secrets_manager.py ......................................                                                                                                                               [100%]

===================================================================================== 58 passed in 103.01s (0:01:43) =====================================================================================
```

Reply:
Based on the comments in #27 where it was added, it's a nice thing to have to spin up a temporary DB per test. I think we should leave it in and correct the import path (`from prefect.test.utilities import prefect_test_harness`).